express-hello-world/server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+const app = express()
+const PORT = 8080
+
+app.use((req, res) => {
+  console.log(req.method)
+  res.send('<h1>HOLA MUNDO !!</h2>')
```

Comment:
Please make sure your HTML is valid HTML (opening and closing tags should match).

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️